lib/internal/Magento/Framework/Mail/Template/TransportBuilderByStore.php

```diff
@@ -1,55 +0,0 @@
-<?php
```

Comment:
Please revert removing of this class and mark it as `@deprecated` with a comment why it was deprecated and a `@see` tag what we should use instead.

PS: as part of 2.3 release line I think we can remove this class

Reply:
Done